docs/tools/erc725js/methods/Encoding.md

```diff
@@ -0,0 +1,507 @@
+    },
+  },
+]);
+/**
+{
+  keys: ['0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5'],
+  values: ['0x6f357c6a2404a2866f05e53e141eb61382a045e53c2fc54831daca9d9e1e039a11f739e1696670733a2f2f516d5154716865424c5a466e5155787535524473387441394a746b78665a714d42636d47643973756b587877526d'],
+}
+*/
+```
+
+```javascript
+myErc725.encodeData([
+  {
+    keyName: 'LSP1UniversalReceiverDelegate',
+    value: '0x1183790f29BE3cDfD0A102862fEA1a4a30b3AdAb',
+  },
+]);
+/**
+{
+  keys: ['0x0cfc51aec37c55a4d0b1a65c6255c4bf2fbdf6277f3cc0730c45b828b6db8b47'],
+  values: ['0x1183790f29be3cdfd0a102862fea1a4a30b3adab'],
+}
+*/
+```
+
+</details>
+
+<details>
+    <summary>Encode a <code>VerifiableURI</code> with hash function, hash and uploaded URL</summary>
+
+```javascript title="Encode a VerifiableURI with hash function, hash and uploaded URL"
+myErc725.encodeData([
+  {
+    keyName: 'LSP3Profile',
+    value: {
+      verification: {
+        method: 'keccak256(utf8)',
+        data: '0x820464ddfac1bec070cc14a8daf04129871d458f2ca94368aae8391311af6361',
+      },
+      url: 'ipfs://QmYr1VJLwerg6pEoscdhVGugo39pa6rycEZLjtRPDfW84UAx',
+    },
+  },
+]);
+/**
+{
+  keys: ['0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5'],
+  values: ['0x6f357c6a820464ddfac1bec070cc14a8daf04129871d458f2ca94368aae8391311af6361696670733a2f2f516d597231564a4c776572673670456f73636468564775676f3339706136727963455a4c6a7452504466573834554178'],
+}
+*/
+```
+
+</details>
+
+<details>
+    <summary>Encode dynamic keys</summary>
+
+```javascript title="Encode dynamic keys"
+const schemas = [
+  {
+    name: 'DynamicKey:<address>',
+    key: '0x0fb367364e1852abc5f20000<address>',
+    keyType: 'Mapping',
+    valueType: 'bytes',
+    valueContent: 'Address',
+  },
+];
+
+myErc725.encodeData(
+  [
+    {
+      keyName: 'DynamicKey:<address>',
+      dynamicKeyParts: ['0xbedbedbedbedbedbedbedbedbedbedbedbedbedb'],
+      value: '0xcafecafecafecafecafecafecafecafecafecafe',
+    },
+  ],
+  schemas,
+);
+/**
+{
+  keys: ['0x0fb367364e1852abc5f20000bedbedbedbedbedbedbedbedbedbedbedbedbedb'],
+  values: ['0xcafecafecafecafecafecafecafecafecafecafe]
+}
+*/
+
+const schemas = [
+  {
+    name: 'DynamicKey:<bytes4>:<string>',
+    key: '0xForDynamicKeysThisFieldIsIrrelevantAndWillBeOverwriten',
+    keyType: 'Mapping',
+    valueType: 'bytes',
+    valueContent: 'Address',
+  },
+];
+
+myErc725.encodeData(
+  [
+    {
+      keyName: 'DynamicKey:<bytes4>:<string>',
+      dynamicKeyParts: ['0x11223344', 'Summer'],
+      value: '0xcafecafecafecafecafecafecafecafecafecafe',
+    },
+  ],
+  schemas,
+);
+/**
+{
+  keys: ['0x0fb367364e1852abc5f2000078c964cd805233eb39f2db152340079088809725'],
+  values: ['0xcafecafecafecafecafecafecafecafecafecafe']
+}
+*/
+```
+
+</details>
+
+<details>
+    <summary>Encode multiple keys at once</summary>
+
+```javascript title="Encode multiple keys at once"
+myErc725.encodeData([
```

Comment:
I'm not sure why there are three elements in the array to encode and in the results there are 5 elements in the key and value array?